elhmn/ckp
# feat: use config file to set code editor

#### This pull request allow users to set their own code editor in a config.yaml file

**Why ?**
People would be able to use vim, but will struggle to quit and we don't want that to happen.

**How ?**
Users can now set an `editor` field in a `.ckp/config.yaml` file.

**Steps to verify:**
create a file in `~/.ckp/config.yaml` then set `editor: nano`

```
editor: nano
``` 

Run the `ckp add code` command and make sure that nano gets opened
